csharp/test/Drivers/Interop/Snowflake/SnowflakeTestingUtils.cs

```diff
@@ -133,11 +133,11 @@ SnowflakeTestConfiguration testConfiguration
         /// Parses the queries from resources/SnowflakeData.sql
         /// </summary>
         /// <param name="testConfiguration"><see cref="SnowflakeTestConfiguration"/></param>
-        internal static string[] GetQueries(SnowflakeTestConfiguration testConfiguration)
+        internal static string[] GetQueries(SnowflakeTestConfiguration testConfiguration, string sqlFilePath = "resources/SnowflakeData.sql")
         {
             StringBuilder content = new StringBuilder();
 
-            string[] sql = File.ReadAllLines("resources/SnowflakeData.sql");
+            string[] sql = File.ReadAllLines(sqlFilePath);
```

Comment:
I guess this is fine, but can we consider making this file an embedded test resource instead of read from the file system? This is currently sensitive to the what the current working directory is.

Reply:
converted to an embedded resource file and added error handling.